src/probnum/linops/_arithmetic.py

```diff
@@ -57,6 +94,295 @@ def matmul(op1: LinearOperator, op2: LinearOperator) -> LinearOperator:
 _mul_fns: _BinaryOperatorRegistryType = {}
 _matmul_fns: _BinaryOperatorRegistryType = {}
 
+########################################################################################
+# Fill Arithmetics Registries
+########################################################################################
+
+# Scaling
+def _mul_scalar_scaling(scalar: ScalarArgType, scaling: Scaling) -> Scaling:
+    if scaling.is_isotropic:
+        return Scaling(scalar * scaling.scalar, shape=scaling.shape)
+
+    return Scaling(scalar * scaling.factors, shape=scaling.shape)
+
+
+def _mul_scaling_scalar(scaling: Scaling, scalar: ScalarArgType) -> Scaling:
```

Comment:
This sort of 'reversed' method can be avoided by using a utility method similar to https://github.com/probabilistic-numerics/probnum/blob/2dadf9787357c5e3a54ed07fa466427fefc54185/src/probnum/randvars/_arithmetic.py#L103